ocs-ktc/src/test/kotlin/org/ostelco/prime/ocs/parser/UserLocationTest.kt

```diff
@@ -30,6 +30,18 @@ class UserLocationTest {
         assertEquals("16", userLocation?.mnc)
     }
 
+    @Test
+    fun userLocationParserMalaysiaSai() {
+
+        val locationHexString = "0105F26182BF804E"
+
+        val userLocation = UserLocationParser.getParsedUserLocation(locationHexString.hexStringToByteArray())
+
+        assertEquals("1", userLocation?.geographicLocationType)
+        assertEquals("502", userLocation?.mcc)
```

Comment:
MCC= malayisia, MNC = digi?   If true, then  be stated in comment

Reply:
It is in the name of the test. I can add Digi as well